commands/provider/assets/readme_template.md

```diff
@@ -23,9 +23,24 @@ You can install MESG Core by running the following command or [follow the instal
 bash <(curl -fsSL https://mesg.com/install)
 ```
 
-## Service
+{{if .Sid}}
+## Purchase the service
```

Comment:
This is an optional step, I would keep it simple with just the link of the website with something like.

To know how to deploy this service please follow the steps on this link https://marketplace.mesg.com/services/sid.

Can find a better text but something like that.